test/integration/parachain/staging/sequential/refund.test.ts

```diff
@@ -33,42 +50,60 @@ describe("refund", () => {
-        vault_3_id = newVaultId(api, vault_3.address, collateralCurrency, wrappedCurrency);
+        vault_3_ids = collateralCurrencies
+            .map(collateralCurrency => newVaultId(api, vault_3.address, collateralCurrency, wrappedCurrency));
     });
 
     after(async () => {
         api.disconnect();
     });
 
     it("should not generate a refund request", async () => {
-        const issueResult = await issueSingle(
-            interBtcAPI,
-            bitcoinCoreClient,
-            userAccount,
-            newMonetaryAmount(0.00005, wrappedCurrency, true),
-            vault_3_id,
-            false,
-            false
-        );
-        assert.isRejected(interBtcAPI.refund.getRequestByIssueId(issueResult.request.id));
-    }).timeout(1000000);
+        for (const vault_3_id of vault_3_ids) {
+            const currencyTicker = currencyIdToMonetaryCurrency(vault_3_id.currencies.collateral).ticker;
+            const issueResult = await issueSingle(
+                interBtcAPI,
+                bitcoinCoreClient,
+                userAccount,
+                newMonetaryAmount(0.00005, wrappedCurrency, true),
+                vault_3_id,
+                false,
+                false
+            );
+            assert.isRejected(
+                interBtcAPI.refund.getRequestByIssueId(issueResult.request.id), 
+                `Expected rejection for refund request with vault 3 (${currencyTicker})`
+            );
+        }
+    }).timeout(2000000);
 
     it("should generate a refund request", async () => {
-        const issueResult = await issueSingle(
-            interBtcAPI,
-            bitcoinCoreClient,
-            userAccount,
-            newMonetaryAmount(0.000012, wrappedCurrency, true),
-            vault_3_id,
-            true,
-            true
-        );
-        const refund = await interBtcAPI.refund.getRequestByIssueId(issueResult.request.id);
-        assert.notEqual(refund.amountBtc.toString(), "0");
-        const refundRequests = await interBtcAPI.refund.list();
-        assert.equal(refundRequests.length, 1);
-    }).timeout(1000000);
+        for (const vault_3_id of vault_3_ids) {
+            const currencyTicker = currencyIdToMonetaryCurrency(vault_3_id.currencies.collateral).ticker;
+            const issueResult = await issueSingle(
+                interBtcAPI,
+                bitcoinCoreClient,
+                userAccount,
+                newMonetaryAmount(0.000012, wrappedCurrency, true),
+                vault_3_id,
+                true,
+                true
+            );
+            const refund = await interBtcAPI.refund.getRequestByIssueId(issueResult.request.id);
+            assert.notEqual(
+                refund.amountBtc.toString(), 
+                "0", 
+                `Expected non-zero amount for refund request with vault 3 (${currencyTicker})`
+            );
+            const refundRequests = await interBtcAPI.refund.list();
```

Comment:
The length of this list should be checked after the for-loop ends, with an expected length of at least `vault_3_ids.length`. We can merge without this change though, since the assert on the line above already checks this essentially.

Reply:
Good spot, thanks. I have a follow-up task to address this.